### PR TITLE
fix(env): update required node engine to v18.18

### PR DESCRIPTION
**Pull request by OnkarRuikar**
- porting from yari PR https://github.com/mdn/yari/pull/11373

### CODE REVIEW

File: .nvmrc [hunk 1]
```diff
@@ -1 +1 @@
-v18.17
```
Comment (bsmth) [inline .nvmrc:1]: Should we be mirroring Yari here https://github.com/mdn/yari/blob/main/.nvmrc

File: .nvmrc [hunk 2]
```diff
@@ -1 +1 @@
...
+v18.18
```
Comment (OnkarRuikar) [inline .nvmrc:1]:
```suggestion
v18
```

File: .nvmrc (hunk 1 above)
Comment (bsmth) [reply .nvmrc:1]: > But `v18` is required by GH workflows.

OK, let's go for it. I guess we're not going to be tinkering with nvm for long given the incoming backend changes.

**Review by bsmth — APPROVED** [review]
TY

**Pull request merged**